src/Chapter1.hs

```diff
@@ -556,7 +559,8 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel c = c `elem` "aeouiAEOUI"
```

Comment:
Absolutely correct! 👍🏼 

One note in here, that sometimes, `elem` could be slower than the explicit pattern matching. I remember there were some benchmarks on one particular case, that showed how moving to pattern matching on each case separately drastically decrease time 🐎

Reply:
Do you mean one guard for every vowel letter? Or something like `| c == 'a' || c == 'e' ...` ?

Reply:
I guess both solutions are equivalent :+1: